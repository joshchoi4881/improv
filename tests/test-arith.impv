func int add(int x, int y) {
  return x + y;
}

func int sub(int x, int y) {
  return x - y;
}

func int mul(int x, int y) {
  return x * y;
}

func int div(int x, int y) {
  return x / y;
}

func int mod(int x, int y) {
  return x % y;
}

func int main() {
  printi(add(4115, 42));
  printi(sub(4115, 42));
  printi(mul(4115, 42));
  printi(div(4115, 42));
  printi(mod(4115, 42));
  return 0;
}
