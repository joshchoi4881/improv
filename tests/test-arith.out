4157
4073
172830
97
41